src/bootchooser.c

```diff
@@ -1654,6 +1702,21 @@ static gboolean custom_set_primary(RaucSlot *slot, GError **error)
 	return custom_backend_set("set-primary", slot->bootname, NULL, error);
 }
 
+/* Get current booted bootname */
+gchar *r_boot_get_bootname(RaucConfig *config)
+{
+	gchar *res = NULL;
+
+	g_return_val_if_fail(config, NULL);
+
+	if (g_strcmp0(config->system_bootloader, "custom") == 0) {
+		res = custom_backend_get_bootname(config);
+	}
+
+	return res;
+}
+
+/* Get state of current slot */
```

Comment:
Unrelated change, but I would be fine with it if you replace 'current' with 'given' to avoid confusion with the above's 'current'.

Reply:
Okay, I have split the commit in two.

Reply:
and replace the word at two places.